PowerShellSyntax.tmLanguage

```diff
@@ -9,7 +9,7 @@
 		<string>psd1</string>
 	</array>
 	<key>name</key>
-	<string>PowerShell</string>
+	<string>Powershell</string>
```

Comment:
Ah.. that was just a visual help for me when working with the definitions in Sublime Text, as I had two versions of the language files installed :) I'll revert it back.